### DISCUSSION TITLE
Set the due time using summary

### DISCUSSION THREAD

**Discussion by hermann-osi** [TechbeeAT/jtxBoard#1196] [Q&A]
Hello,
I like this app but I wonder if it's possible to set the due date/time from the summary like "restaurant today at 13h" or "meeting friday at 10h30" ?
I looked around and I could not find this possibility yet.

Thank you in advance

**Comment by patrickunterwegs** [comment]
Hi @hermann-osi , 
sorry, no, this is currently not possible. I have thought about functionalities like that, but it's hard to interpret the text in such a way - especially if it comes to support for multiple languages. I think some AI stuff could help here, but I would need some time to investigate what could be possible.

**Reply by stark-bit** [reply]
Hey, I'm also interested in this feature. What about delegating the trigger to the users?

Set due date to tomorrow → @tomorrow / @tom / @morgen
Set due date N hours from now → @in-3h / @in3h / @3h15m
etc.

No NLP needed - the syntax is unambiguous and language-agnostic.

**Comment by hermann-osi** [comment]
Ah I understand. Indeed it would be a lot to add, nevermind then. 
Do you know if there is a way to quickly set the time for start and due instead ? Currently, I have to set the same time twice in a specific order if I want them to be similar. (I have some app triggering with the start time like thunderbird and other app triggered with due time) Maybe there is a way to do it in one go.

**Reply by patrickunterwegs** [reply]
Hi @hermann-osi , 
You can set a default start and/or due date when you create a new task in the settings. Is it that what you mean?

![Screenshot_20240203_043728_jtx Board.jpg](https://github.com/TechbeeAT/jtxBoard/assets/72232737/ccea4e86-19e1-4cd5-ba19-332b619fe19b)
